modules/tinymce/src/plugins/help/test/ts/browser/CustomTabsTest.ts

```diff
@@ -1,149 +1,152 @@
-import { Chain, Logger, NamedChain, Pipeline, UiFinder } from '@ephox/agar';
-import { assert, UnitTest } from '@ephox/bedrock-client';
+import { UiFinder } from '@ephox/agar';
+import { before, describe, it } from '@ephox/bedrock-client';
 import { Arr } from '@ephox/katamari';
 import { Editor as McEditor } from '@ephox/mcagar';
-import { Html, SugarElement } from '@ephox/sugar';
-import HelpPlugin from 'tinymce/plugins/help/Plugin';
-import Editor from '../../../../../core/main/ts/api/Editor';
-import Theme from '../../../../../themes/silver/main/ts/Theme';
+import { Html, SugarDocument } from '@ephox/sugar';
+import { assert } from 'chai';
 
-UnitTest.asynctest('Custom Help Tabs test', (success, failure) => {
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/help/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
 
-  HelpPlugin();
-  Theme();
-
-  const doc = SugarElement.fromDom(document);
-
-  const compareTabNames = (expectedNames: string[]) => Chain.op((editor: Editor) => {
-    editor.execCommand('mceHelp');
-    const actualTabs = UiFinder.findAllIn(doc, 'div.tox-dialog__body-nav-item.tox-tab');
-    const actualNames: string[] = Arr.map(actualTabs, (tab) => Html.get(tab));
-    Arr.map(expectedNames, (x, i) => assert.eq(x, actualNames[i], `Tab names did not match. Expected: ${expectedNames}. Actual: ${actualNames}`));
+describe('browser.tinymce.plugins.help.CustomTabsTest', () => {
+  before(() => {
+    Plugin();
+    Theme();
   });
 
-  const makeStep = (config: Object, expectedTabNames: string[]) => Chain.asStep({}, [
-    McEditor.cFromSettings(config),
-    NamedChain.asChain([
-      NamedChain.direct(NamedChain.inputName(), Chain.identity, 'editor'),
-      NamedChain.read('editor', compareTabNames(expectedTabNames)),
-      NamedChain.output('editor')
-    ]),
-    McEditor.cRemove
-  ]);
-
-  Pipeline.async({}, [
-    Logger.t('Default help dialog', makeStep({
-      plugins: 'help',
-      toolbar: 'help',
-      theme: 'silver',
-      base_url: '/project/tinymce/js/tinymce'
-    }, [ 'Handy Shortcuts', 'Keyboard Navigation', 'Plugins', 'Version' ])),
+  const compareTabNames = (editor: Editor, expectedNames: string[]) => {
+    editor.execCommand('mceHelp');
+    const actualTabs = UiFinder.findAllIn(SugarDocument.getDocument(), 'div.tox-dialog__body-nav-item.tox-tab');
+    const actualNames = Arr.map(actualTabs, (tab) => Html.get(tab));
+    Arr.map(expectedNames, (x, i) => {
+      assert.equal(x, actualNames[i], `Tab names did not match. Expected: ${expectedNames}. Actual: ${actualNames}`);
```

Comment:
Won't the "Expected, actual" be included in the error message just thanks to chai?

Reply:
Yeah, that's wrong...

Reply:
Fixed